R/data_frame.R

```diff
@@ -32,7 +32,6 @@ is_data_frame <- function(df,
                           exact_colnames = NULL,
                           required_rownames = NULL,
                           required_colnames = NULL,
-                          allow_na = FALSE,
```

Comment:
now, technically this breaks backward compat, so we should have a deprecation process, but I am willing to take the risk because I am 99.999% certain I have not used it, and if I did, it will be easy to spot.